### PR TITLE
[prometheus-blackbox-exporter] add action=replace to metricsRelabelings

### LINKED ISSUE #3626: [prometheus-snmp-exporter] GitOps keeps getting OutOfSync due to missing param in ServiceMonitor endpoint
### Describe the bug a clear and concise description of what the bug is.

It looks like the action parameter is missing here.
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-snmp-exporter/templates/servicemonitor.yaml#L29

image

### What's your helm version?

ArgoCD 2.7.8

### What's your kubectl version?

ArgoCD 2.7.8

### Which chart?

prometheus-snmp-exporter

### What's the chart version?

1.6.0

### What happened?

_No response_

### What you expected to happen?

_No response_

### How to reproduce it?

_No response_

### Enter the changed values of values.yaml?

_No response_

### Enter the command that you execute and failing/misfunctioning.

https://github.com/GiganticMinecraft/seichi_infra/blob/main/seichi-onp-k8s/manifests/seichi-kubernetes/apps/cluster-wide-apps/app-of-other-apps/prometheus-snmp-exporter.yaml

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by thinkmassive**
#### What this PR does / why we need it

This (default) parameter is automatically added by k8s, causing the in-cluster object to differ from the manifest created by this chart. This discrepancy causes GitOps tooling to consider the resource out of date.

#### Which issue this PR fixes
- fixes #3626

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by thinkmassive** [comment]
Hello maintainers, does this small change need anything else for approval? @monotek

**Comment by thinkmassive** [comment]
Seeking any feedback I can get to push this contribution forward. Thanks in advance @desaintmartin @gianrubio @rsotnychenko

**Review by monotek — APPROVED** [review]
Let me guess: It's argocd again? :/

**Pull request merged**
